src/ORM/Timestampable/TimestampableSubscriber.php

```diff
@@ -43,42 +33,27 @@ public function loadClassMetadata(LoadClassMetadataEventArgs $eventArgs)
             return;
         }
 
-        if ($this->isTimestampable($classMetadata)) {
-            if ($this->getClassAnalyzer()->hasMethod($classMetadata->reflClass, 'updateTimestamps')) {
-                $classMetadata->addLifecycleCallback('updateTimestamps', Events::prePersist);
-                $classMetadata->addLifecycleCallback('updateTimestamps', Events::preUpdate);
-            }
+        if (is_subclass_of($classMetadata->getName(), 'Knp\DoctrineBehaviors\Model\Timestampable\TimestampableInterface')) {
```

Comment:
Wouldn't you prefer basing yourself on the *Properties trait to trigger the mapping ? Using the interface might cause problems in case of inheritance (CF #317).

What do you think about the pattern I used in my PRs (e.g. https://github.com/KnpLabs/DoctrineBehaviors/pull/322/files#diff-9af172be101c124fe7673df513efc25aL296)